server/jetstream_cluster.go

```diff
@@ -7276,6 +7276,10 @@ func (s *Server) jsClusteredConsumerRequest(ci *ClientInfo, acc *Account, subjec
 		cfg.MaxAckPending = JsDefaultMaxAckPending
 	}
 
+	if cfg.PinnedTTL == 0 {
+		cfg.PinnedTTL = JsDefaultPinnedTTL
```

Comment:
Should this be with the other check in `checkConsumerCfg`? Placement here feels weird.

Reply:
It should be in `setConsumerConfigDefaults`, and it actually already is there, so I removed the duplicate.